app/src/test/java/com/app/calculator/OperationPresenterTest.java

```diff
@@ -10,4 +10,18 @@
  * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
  */
 public class OperationPresenterTest {
```

Comment:
The asserts found in this class doesn't let you know if the method in the presenter are correct. Consider testing the method from the presenter to ensure that it works properly. If needed, you can make a refactor so the presenter is able to be tested.